diff --git a/include/target.mk b/include/target.mk
index 1234029..85b863c 100644
--- a/include/target.mk
+++ b/include/target.mk
@@ -26,9 +26,7 @@ DEFAULT_PACKAGES:=\
 	uclient-fetch \
 	urandom-seed \
 	luci luci-compat wget-ssl curl ca-certificates htop \
-	default-settings luci-app-upnp luci-app-wol luci-app-vlmcsd luci-app-ramfree \
-	luci-app-ddns ddns-scripts-cloudflare ddns-scripts_aliyun ddns-scripts_dnspod \
-	luci-app-timecontrol luci-app-control-timewol luci-app-control-webrestriction luci-app-control-weburl \
+	default-settings \
 	urngd
 
 ifneq ($(CONFIG_SELINUX),)
