diff --git a/target/linux/ipq40xx/Makefile b/target/linux/ipq40xx/Makefile
index 22a619e..c0e869d 100644
--- a/target/linux/ipq40xx/Makefile
+++ b/target/linux/ipq40xx/Makefile
@@ -20,7 +20,6 @@ DEFAULT_PACKAGES += \
 	kmod-leds-gpio kmod-gpio-button-hotplug swconfig \
 	kmod-ath10k-ct wpad-basic \
 	kmod-usb3 kmod-usb-dwc3 ath10k-firmware-qca4019-ct \
-	autocore-arm automount autosamba \
-	ca-certificates htop fdisk e2fsprogs
+	autocore-arm automount ca-certificates
 
 $(eval $(call BuildTarget))
