diff --git a/package/lean/automount/Makefile b/package/lean/automount/Makefile
index 7bcd20079..cd9b7800c 100644
--- a/package/lean/automount/Makefile
+++ b/package/lean/automount/Makefile
@@ -31,8 +31,8 @@ endef

 define Package/automount/install
	$(INSTALL_DIR) $(1)/etc/uci-defaults
-	$(INSTALL_BIN) ./files/15-automount $(1)/etc/15-automount
-	$(INSTALL_BIN) ./files/zzz-move-automount $(1)/etc/uci-defaults/zzz-move-automount
+	$(INSTALL_DIR) $(1)/etc/hotplug.d/block
+	$(INSTALL_BIN) ./files/15-automount $(1)/etc/hotplug.d/block/15-automount
 endef

 $(eval $(call BuildPackage,automount))
